lib/trie/trie_test.go

```diff
@@ -11,11 +11,22 @@ import (
 
 	"github.com/ChainSafe/gossamer/internal/trie/node"
 	"github.com/ChainSafe/gossamer/lib/common"
-	gomock "github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func Test_EmptyHash(t *testing.T) {
+	t.Parallel()
+
+	expected := common.Hash{
+		0x3, 0x17, 0xa, 0x2e, 0x75, 0x97, 0xb7, 0xb7,
+		0xe3, 0xd8, 0x4c, 0x5, 0x39, 0x1d, 0x13, 0x9a,
+		0x62, 0xb1, 0x57, 0xe7, 0x87, 0x86, 0xd8, 0xc0,
+		0x82, 0xf2, 0x9d, 0xcf, 0x4c, 0x11, 0x13, 0x14,
+	}
```

Comment:
note: that was taken from the `EmptyHash` variable value before these PR changes